components/results/ResultsFailure.tsx

```diff
@@ -0,0 +1,11 @@
+import CenteredCardContent from '../common/CenteredCardContent';
+
+import ResultsRetryButton from './ResultsRetryButton';
+
+export default function ResultsFailure(): JSX.Element {
+  return (
+    <CenteredCardContent title="Verification failed" description="Sorry, we invite you to try again the verification." iconClasses="fa fa-times-circle text-danger mb-4">
```

Comment:
What's the difference between ResultFailure and ResultsError? https://github.com/NEARFoundation/near-kyc/pull/10/files#r990093777

The error messages look almost the same but not quite. Should they be identical?

Oh, I see that the titles differ.

So maybe the message in this one should say:

> We could not verify your identity. If you believe that you submitted any information incorrectly, you may try again.

or something like that.

Reply:
Error is more due to a bug user do not exist, or backend error.

Failure is when the validation didn't pass

Reply:
Thanks for your text it's indeed more clear!